mvvm-livedata/src/nonAndroidMain/kotlin/dev/icerock/moko/mvvm/livedata/LiveData.kt

```diff
@@ -11,7 +11,7 @@ actual open class LiveData<T>(initialValue: T) {
     actual open val value: T
         get() = storedValue
 
-    actual fun addObserver(observer: (T) -> Unit) {
+    actual open fun addObserver(observer: (T) -> Unit) {
```

Comment:
if we open `addObserver` so we should open and `removeObserver` too. and do same in `androidMain` too